lib/qam.pm

```diff
@@ -127,9 +127,15 @@ sub remove_test_repositories {
 
 sub advance_installer_window {
     my ($screenName) = @_;
+    my $build = get_var('BUILD');
 
     send_key $cmd{next};
     die 'Unable to create repository' if check_screen('unable-to-create-repo', 5);
+    if ($build =~ m/^MR:/) {
+        if (check_screen("gpg-key-check", 30)) {
```

Comment:
30 seconds is pretty much

Reply:
Please use existing tag ` import-untrusted-gpg-key` https://openqa.suse.de/tests/5671169#step/scc_registration/23

Reply:
existing tag looks like it was created with **exact** keys  in mind, unfortunately for MR is key changing for every repo

Reply:
no, exact keys have exact tag https://openqa.suse.de/tests/5671169/modules/scc_registration/steps/23/edit
`import-untrusted-gpg-key` is "universal"
Known keys have tag `import-known-untrusted-gpg-key` https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/utils.pm#L207

Reply:
ahh, thx